### PR TITLE
Lesson 1

### PR DESCRIPTION
**Pull request by kseniyabuko**

### CODE REVIEW

**Comment by GlebLitvinov** [comment]
Lesson1.java looks good, please remove all files under .idea directory from VCS, though (misc.xml, modules.xml e.t.c)

File: .idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (GlebLitvinov) [inline .idea/misc.xml]: you need to remove this file and all files under .idea folder from vcs. Try reading this page(https://stackoverflow.com/questions/36163781/remove-file-from-version-control-in-intellij-idea) or find a solution by yourself

**Review by GlebLitvinov — CHANGES_REQUESTED** [review]

File: .idea/misc.xml (hunk 1 above)
Comment (kseniyabuko) [reply .idea/misc.xml]: Thanks, done.
Shall I merge pull request now after you approve?

**Review by GlebLitvinov — APPROVED** [review]

File: .idea/misc.xml (hunk 1 above)
Comment (GlebLitvinov) [reply .idea/misc.xml]: Yes, please. Good job

**Pull request merged**